build/helper/documentation_snippets.py

```diff
@@ -96,6 +96,14 @@
 { 'simulate': True, 'driver_setup': { 'Model': '<model number>',  'BoardType': '<type>' } }
 '''
 
+default_close_function_doc = '''
+Closes the driver session and cleans up.
```

Comment:
Should we add: "After calling this, the Session object is no longer valid and cannot be used."

Reply:
Yes, that seems like a good thing to add.